Checkout/design/adminhtml/default/default/template/bold/checkout/form/field/onboarding/banner.phtml

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
When testing these changes locally, the banner doesn't render due to the error `Not valid template file:adminhtml/base/default/template/bold/checkout/form/field/onboarding/banner.phtml`

Might be specific to my local setup but wanted to bring it up - maybe we could pair on comparing envs/resolving?

Reply:
> Perhaps we should mark this as a draft PR

Good idea about marking this PR as a draft.

> ...maybe we could pair on comparing envs/resolving?

Yes, let's set that up for later this morning.

Reply:
The errors I was seeing were due to local issues on my end, Joseph helped resolve these. Tested locally and confirmed it works great!